src/class-wpml-elementor-translatable-nodes.php

```diff
@@ -74,7 +74,11 @@ public function get( $node_id, $element ) {
 				if ( isset( $node_data['integration-class'] ) ) {
 					foreach ( $this->get_integration_classes( $node_data ) as $class ) {
 						try {
-							$node    = new $class();
+							if($class instanceof \WPML_Elementor_Module_With_Items){
```

Comment:
@lxbdr, could you please add spaces around the parenthesis: `if ( $class instanceof \WPML_Elementor_Module_With_Items ) {`.

We are trying to follow the WP coding standards (https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#brace-style).